_projects/2022/100376720/100376720.Rmd

```diff
@@ -0,0 +1,401 @@
+
+```{r}
+
+educ <- df %>%
+  drop_na(Downward_Mobility) %>% 
+  filter(Country != "Northern Ireland" &
+         Country != "England" &
+         Country != "Sub-national entities" &
+         Country != "Partners" &
+         Country != "Average"
+  ) %>%
+  pivot_longer(cols = ends_with("ility"), 
+               names_to = "Mobility_type", 
+               values_to = "Mobility_perc") 
+
+df_order <- educ[educ$Mobility_type == "Upward_mobility", ]
+df_order2 <- educ[educ$Mobility_type == "Status_quo_Mobility",]
+df_order3 <- educ[educ$Mobility_type == "Downward_Mobility",]
+
+```
+
+
+## First graph
+
+The code for this visual representation can be divided into six parts:
+
+1. Plotting the data as vertical columns ("stack") and arranging it in descending order based on the Upward_mobility category.
+2. Changing the theme by removing elements such as axis.ticks, axis.texts, and background.
+3. Adding numbers to the graph using geom_text and adjusting their positions and formatting (e.g., adding a percent symbol or adding extra space).
+5. Adding a subtitle and adjusting its position and margin to make room for annotations.
+6. Adding annotations, including both text annotations and line ("segment") annotations.
+7. Adjusting the plot coordinates to make it easier to combine it with the second graph.
+
+
+```{r}
+
+p = educ %>%
+  group_by(Country, Mobility_perc) %>%
+  ggplot() +
+  aes(Mobility_perc, reorder(Country, Mobility_perc)) +
+  geom_col(data = df_order, position = "stack") +
+  scale_fill_manual(values = c("#ffc96f", "#b8bdd0", "#5e4c5f")) + 
+  geom_col(data = educ[educ$Mobility_type != "Upward_mobility", ], position = "stack") +
+  geom_col(aes(fill = Mobility_type)) + 
+  labs(y = NULL) +
+  theme(legend.position = "none",
+        axis.title.x = element_blank(),
+        axis.text.x = element_blank(),
+        axis.ticks.x = element_blank(),
+        axis.ticks.y = element_blank(),
+        axis.text.y = element_text(hjust = 0),
+        plot.background = element_rect(fill = "white", colour = NA), 
+        panel.background = element_rect(fill = "white", colour = NA),
+        legend.background = element_rect(fill = "white", colour = NA),
+        legend.key = element_rect(fill = "white"))+
+  scale_y_discrete(labels = function(x) ifelse(x == "United States",
+                                                expression(bold("United States")),
+                                                x))+
+ geom_text(data = df_order, 
+            aes(x=12, label = ifelse(Country == "South Korea", paste0(Mobility_perc, "%"), 
+                                     ifelse(Country != "South Korea", paste0(Mobility_perc, "    "), Mobility_perc))), size=3,
```

Comment:
There are some lines that are too wide in this chunk. See [the preview](https://deploy-preview-3--mucss-dataviz.netlify.app/_projects/2022/100376720/100376720). Please, make them shorter, typically a maximum of 80 characters.